distributed/src/main/java/org/infinispan/tutorial/simple/distributed/InfinispanDistributed.java

```diff
@@ -14,13 +15,16 @@
    public static void main(String[] args) throws Exception {
       // Setup up a clustered cache manager
       GlobalConfigurationBuilder global = GlobalConfigurationBuilder.defaultClusteredBuilder();
-      // Make the default cache a distributed synchronous one
+      // Initialize the cache manager
+      DefaultCacheManager cacheManager = new DefaultCacheManager(global.build());
+      //Create cache configuration
       ConfigurationBuilder builder = new ConfigurationBuilder();
       builder.clustering().cacheMode(CacheMode.DIST_SYNC);
-      // Initialize the cache manager
-      DefaultCacheManager cacheManager = new DefaultCacheManager(global.build(), builder.build());
-      // Obtain the default cache
-      Cache<String, String> cache = cacheManager.getCache();
+      // Obtain a cache
+      Cache<String, String> cache = cacheManager.administration()
+              .withFlags(CacheContainerAdmin.AdminFlag.VOLATILE)
```

Comment:
The indent here is 2*4 spaces, but in most other places it's 3 spaces. 

I suggest keeping the indent 3 spaces, and for fluent methods I would prefer aligning them on the `.`, e.g.

```java
      Cache<String, String> cache =
            cacheManager.administration().withFlags(CacheContainerAdmin.AdminFlag.VOLATILE)
                        .getOrCreateCache("cache", builder.build());
```

 (In IntelliJ settings, Code Style, Wrapping and Braces, Chained method calls, Align when multiline.)